src/Web/Sfa.Das.Sas.Web.UnitTests/Infrastructure/Web/Helpers/UrlEncoderTests.cs

```diff
@@ -32,10 +38,19 @@ public class UrlEncoderTests
         [TestCase("ST.PATRICK'S INTERNATIONAL COLLEGE LIMITED", "st-patricks-international-college-limited")]
         [TestCase("GUY'S AND ST THOMAS' NHS FOUNDATION TRUST", "guys-and-st-thomas-nhs-foundation-trust")]
         [TestCase("Name with [ weird $ characters", "name-with-\\[-weird-\\$-characters")]
+        [TestCase("Travel Services: Tour Operators - Head Office", "travel-services-tour-operators-head-office")]
+        [TestCase("SemiColon; & Consulting", "semicolon-and-consulting")]
+        [TestCase("Colon: & Consulting", "colon-and-consulting")]
+        [TestCase("Back\\Slash", "back-slash")]
+        [TestCase("Forward/Slash", "forward-slash")]
+        [TestCase("Installation electrician / maintenance electrician", "installation-electrician-maintenance-electrician")]
+        [TestCase("King King", "king-king")]
+        [TestCase("Size? Online", "size-online")]
+        [TestCase("Shazam! Inc.", "shazam-inc")]
         public void ShouldReturnStringModifiedForUrlUsage(string inputText, string encodedText)
```

Comment:
`[TestCase("vessels of less than 3,000 gross tonnage", "vessels-of-less-than-3-000-gross-tonnage")]` is failing

Reply:
good spot. I'll cover that off once I've finished Scott's points.